platforms/documentation/docs/src/docs/userguide/platforms/jvm/java_testing.adoc

```diff
@@ -559,6 +551,53 @@ Users commonly want to run integration tests after the unit tests, because they
 
 For information on how to determine code coverage for tests in additional source sets, see the <<jacoco_plugin#jacoco_plugin,JaCoCo Plugin>> and the <<jacoco_report_aggregation_plugin#jacoco_report_aggregation_plugin, JaCoCo Report Aggregation Plugin>> chapters.
 
+[[sec:java_testing_troubleshooting]]
+== Troubleshooting common problems
+
+=== Tests are not detected or executed
+
+If no tests are detected, Gradle will skip the `Test` task entirely.
+
+There could be a few causes:
+
+ * No test sources exist. This is normal behavior.
+ * Test sources exist, but they were not compiled. The sources may be in the wrong directory or the source set may not be configured correctly.
```

Comment:
```suggestion
 * Test sources exist but are not compiled. This may happen if:
 ** The source files are located in the wrong directory.
 ** The source set is not configured correctly.
```